src/main/java/com/blogify/blogapi/model/CommentReaction.java

```diff
@@ -0,0 +1,39 @@
+package com.blogify.blogapi.model;
+
+import java.util.UUID;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+
+@Entity
+@Table(name = "comment_reaction")
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+public class CommentReaction {
+    @Id
+    private String id;
+
+    /* @ManyToOne
+    @JoinColumn(name = "user_id")
+    private User user;
+    // TODO: Add relation with User, waiting for the completion of the User entity */
+
+    @ManyToOne
+    @JoinColumn(name = "comment_id")
+    private Comment comment;
```

Comment:
L2 - here we have a bi-directional relation between Comment and CommentReaction (Comment contains List of CommentReaction and CommentReaction refers to one comment)
so we need to have an unidirectional relation: only Comment contains List of CommentReaction

so remove this attribute: 
    private Comment comment;